mandatory/2-function-creation.js

```diff
@@ -6,7 +6,9 @@ Complete the function to check if the variable `num` satisfies the following req
 Tip: use logical operators
 */
 
-function validate(num) {}
+function validate(num) {
+  return Number.isFinite(num) && num % 2 === 0 && num <= 100 ? true : false;
```

Comment:
Good job Hamra with using different style of if statement(ternary operator). I learned now. Thanks.

Reply:
agreed, nice ternary. You can also check if `num` is a number by using `typeof`